templates/glossary.md

```diff
@@ -0,0 +1,253 @@
+# Lean Glossary
+
+### abbreviation
+
+In the context of editing Lean files, an abbreviation is a way of
+entering a symbol not generally found on standard keyboard layouts using
+a descriptive shortcut.
+
+For instance, the not-equal symbol "≠" can be entered using the sequence
+`\neq`.
+
+The full list of abbreviations (and their replacements) can be found [in the
+vscode-lean repository](https://github.com/leanprover/vscode-lean/blob/master/src/abbreviation/abbreviations.json).
+
+### attribute
+
+One or more tags or markings which may be applied to a Lean object,
+and which may affect either its behavior or the behavior of other Lean
+objects which interact with it. Attributes may be defined either within
+[core Lean](#core-lean), within mathlib, or within any Lean code.
+
+Applying an attribute to an object is done by prefixing its
+[declaration](#declaration) with `@[name-of-attribute]`, or by using the
+`attribute` statement.
+
+The `@[simp]` attribute, for example, tags an object (ostensibly a `lemma` or
+`theorem`) as being a [simp lemma](#simp-lemma).
+
+For further details, including a list of commonly used attributes, see [this
+section of the Lean reference manual](https://leanprover.github.io/reference/other_commands.html#attributes).
+
+### beta reduction
+
+A specific simplification operation in [dependent type
+theory](#dependent-type-theory) (and Lean's implementation of it) which
+may be performed as part of deciding whether two [terms](#term) are
+[definitionally equal](#defeq).
+
+More precisely, it is the process of simplifying an expression such as
+`(λ x, x) a` by replacing appearances of the variable `x` with `a`.
+
+#### See also
+
+* [Section 2.3 of Theorem Proving in Lean](https://leanprover.github.io/theorem_proving_in_lean/dependent_type_theory.html#function-abstraction-and-evaluation)
+
+### big operator
+
+### binder
+
+### bundled vs unbundled
+
+
+### cache
+
+As the compilation time of mathlib well exceeds 2h on modest computers, we host a *cache* for each commit to the mathlib repository. Each cache consists of [olean files](#olean-files) for all of mathlib.
+
+### carrier
+
+### class
+
+A *class* (aka *typeclass*) is a [structure](#structure) that can be handled by [typeclass inference](#typeclass-inference).
+
+### coercion / ↑
+
+### code linter
+
+A *code linter* is a [linter](#lint) concerned with how code works. Concretely, this is [a collection of Lean programs](https://leanprover-community.github.io/mathlib_docs/tactic/lint/frontend.html) checking that... Errors are flagged in the [nolint file](https://github.com/leanprover-community/mathlib/blob/master/scripts/nolints.txt).
```

Comment:
```suggestion
A *code linter* is a [linter](#lint) concerned with how code works. Concretely, this is [a collection of Lean programs](https://leanprover-community.github.io/mathlib_docs/tactic/lint/frontend.html) checking that.
[TODO]
Errors are flagged in the [nolint file](https://github.com/leanprover-community/mathlib/blob/master/scripts/nolints.txt).
```
Do I understand correctly that `...` indicates that this was unfinished? If so, I suggested inserting the word `TODO`, so that we can quickly spot such places.